src/styles/_base.scss

```diff
@@ -70,26 +73,33 @@ nav {
     padding: .5em 0 .5em .25em;
     margin: 0;
 
-    background-color: white;
+    background-color: $color-gray-cool-2;
     font-size: 1.2em;
     font-weight: 700;
     text-transform: lowercase;
     font-variant: small-caps;
   }
 
   > #nav-scroll-container > section > ul > li {
-    margin-bottom: 1em;
+    margin-bottom: .7em;
```

Comment:
Eh, I like this better with a little more space.

Reply:
I think a full line is too much space though. .8em looks fine, but .9 seems too big